R/generate_simulated_data.R

```diff
@@ -339,41 +348,29 @@ generate_simulated_data <- function(r_in_weeks = # nolint
     )
   }
 
-  log_r_site <- spatial_rt_process(
-    log_r_state_week,
-    corr_function,
-    corr_fun_params,
-    phi_rt,
-    sigma_eps
+  # Using stan exposed functions for forward spatial Rt process.
+  sigma_matrix <- sigma_eps * corr_function(corr_fun_params)
+  spatial_deviation_noise_matrix <- spatial_deviation_noise_matrix_rng(
```

Comment:
I think you want to do a similar thing with the `spatial_deviation_init` (though it doesn't need to call a stan function, just use R to draw from a normal with mean `log_r_state_week` and stdeviation `sigma_rt`

Reply:
Are you referring to the spatial_deviation_init or the whole spatial_deviation_noise_matrix?  I am a bit confused on this comment.

Reply:
I am suggesting that instead of passing in `spatial_deviation_init`, you do the same thing that you do here for `spatial_deviation_noise_matrix`. So to get the initial deviaiton you would do something like 

`spatial_deviation_init <- log_state_Rt_week[,1] + rnorm(n= n_weeks, mean = 0, sd = sigma_rt)`

Reply:
After my one-on-one meeting with @dylanhmorris, we decided to not have an init value for either spatial or aux site rt functions.  For the initial values, we will just use $\text{Normal}\_{n\_{k}}(0,\Sigma)$ and $\text{Normal}(0,\text{ScalingFactor}\times\sigma\_{\epsilon})$ for spatial and aux rt's.  This will be used in the construct functions after passing in a matrix of random vectors for spatial rt and after passing in a vector of random values for aux site.  If we end up deciding to do something different with the initial values of the matrix of random vectors or vector of random values, we can add that change in the construction of these random quantities, within the noise rng functions.

Reply:
I think that makes sense, these changes look good!